src/Uno.UWP/ApplicationModel/Contacts/ContactAddress.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+
+namespace Windows.ApplicationModel.Contacts
+{
+	public partial class ContactAddress
+	{
+		public string StreetAddress { get; set; }
+		public string Region { get; set; }
+		public string PostalCode { get; set; }
+		public string Locality { get; set; }
+		public ContactAddressKind Kind { get; set; }
+		// public string Description { get; set; }

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4900834)

Reply:
I'm still waiting for answer - leave comments about field sizes and unimplemented (yet) members, or remove them.